core/templates/components/ck-editor-helpers/ck-editor-copy-toolbar/ck-editor-copy-toolbar.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="ck-editor-copy-toolbar-container">
```

Comment:
prefix with `oppia-` here and elsewhere*

Reply:
Done